packages/SwingSet/docs/configuration.md

```diff
@@ -133,17 +184,78 @@ not yet supported.
+  debug outputs.  If omitted, it defaults to the property name that was used for
+  the vat descriptor.
+
+- `enableDisavow`, if `true`, adds `vatPowers.disavow()`, which allows vat code
+  to explicitly disavow interest in an imported Presence. This will trigger a
+  `syscall.dropImports` of the associated object ID. By default, this
+  function is not added to `vatPowers`.
+
+- `managerType` indicates what sort of vat worker to run the associated vat in.
+  If omitted, it defaults to the swingset's default manager type.
 
 - `metered` specifies whether the vat should be have metering turned
-  on or not (defaults to `false` if omitted).
+  on or not. Defaults to `false`.
+
+- `reapInterval` specifies the reap interval for this particular vat.  It
+  defaults to the swingset's default reap interval.
 
 - `enableSetup` indicates that the vat module may use the older, lower
   level `setup()` API, which allows a vat to be defined independent of
   the liveslots framework (defaults to `false` if omitted).  (If you
   don't understand what this means, do not use this.)
 
-- `enableDisavow`, if true, adds `vatPowers.disavow()`, which allows vat code
-  to explicitly disavow interest in an imported Presence. This will trigger a
-  `syscall.dropImports` of the associated object ID. By default, this
-  function is not added to `vatPowers`.
+- `critical`, if `true`, marks the vat as critical to the correct functioning of
+  the swingset.  If a critical vat suffers a failure, the kernel will panic and
+  the swingset as a whole will be terminated without saving any intermediate
+  state that resulted during the crank that caused the vat to die.  Defaults to
+  `false`.
+
+- `useTranscript`, if `true`, says the vat should record a transcript of all
+  deliveries made to it so that these can be replayed at a later time to
+  reconstruct the internal state of the vat.  Defaults to `true`.
+
+- `virtualObjectCacheSize` tells the vat's virtual object manager how many
+  virtual objects should have their state kept live in memory at any given
+  time.  Defaults to an implementation defined value.
+
+## Dynamic option setting
+
+Some swingset and vat configuration options can be dynamically updated within a
+running swingset.
+
+### Dynamic kernel options
+
+Swingset-wide (kernel) options are updated by calling the
+`changeKernelOptions(options)` method on the swingset's controller object, where
+`options` is an object indicating which options to change and the values to
+change them to.
+
+Currently supported kernel options that you can change this way are:
+
+- `defaultReapInterval`
+
+- `snapshotInterval`
+
+### Dynamic vat options
+
+The options for a single vat may be updated by sending the `changeOptions(options)`
+message to the vat's admin node (note that because the admin node is involved,
+dynamic options update is not available for statically configured vats, i.e.,
+vats configured using the config object described above).
+
```

Comment:
lets make the example a bit more precise, maybe ".. updated by using `E(adminFacet).changeOptions(options)` to send a `changeOptions` message to the particular vat's admin facet. Note that only dynamic vats have admin facets, so .."